watcher/watcher.go

```diff
@@ -103,7 +121,7 @@ func (watcher Watcher) watchLogShiftOuts(parent context.Context) {
 		copy(nonceBytes[:], pack.NewU256FromU64(pack.NewU64(nonce)).Bytes())
 
 		// Send the burn transaction to the resolver.
-		params, err := watcher.burnToParams(pack.NewU256FromU64(pack.NewU64(amount)), pack.String(to), nonceBytes)
+		params, err := watcher.burnToParams(iter.Event.Raw.TxHash.Bytes(), pack.NewU256FromU64(pack.NewU64(amount)), pack.String(to), nonceBytes, watcher.gpubkey)
 		if err != nil {
 			watcher.logger.Infof("[watcher] cannot get params from burn transaction: %v", err)
 		}
```

Comment:
`continue` if we get an error.
Also, I'm thinking if we should use `logger.Errorf` here and below.